src/test/scala/edu/ie3/simona/model/participant2/HpModelSpec.scala

```diff
@@ -526,10 +764,102 @@ class HpModelSpec
               minPower should approximate(Kilowatts(expectedMinPower))
               maxPower should approximate(Kilowatts(expectedMaxPower))
           }
+      }
+    }
+
+    "Handle operating correctly" in {
```

Comment:
```suggestion
    "determine operating point without flex control correctly" in {
```

Reply:
Regarding capital first letter: Just realized you use a different naming structure. It was supposed to work like this:

`HpModel` should `determine whatever` in {}

But Intellij doesn't display it like this, so to me both is fine